doc/source/tune-usage.rst

```diff
@@ -490,12 +492,13 @@ Here are a few examples of command line calls.
     | MyTrainableClass | 3_height=54,width=21  | 7b99a28a   |
     +------------------+-----------------------+------------+
     Dropped columns: ['status', 'last_update_time']
+    Please increase your terminal size to view remaining columns.
 
-- ``tune list-experiments``: List tabular information about experiments within a project. Add the ``--sort`` flag to sort the output by specific columns. Add the ``--filter`` flag to filter the output in the format ``"<column> <operator> <value>"``.
+- ``tune list-experiments``: List tabular information about experiments within a project. Empty columns will be dropped by default. Additional columns may be dropped in order for the output to fit within the user's terminal window. Add the ``--sort`` flag to sort the output by specific columns. Add the ``--filter`` flag to filter the output in the format ``"<column> <operator> <value>"``. Add the ``--output`` flag to write the trial information to a specific file (CSV or Pickle).
```

Comment:
Maybe we can be a little less verbose about the terminal viewing? The text in the example is already pretty clear; we can probably just keep `Empty columns will be dropped by default.` (for both `list-experiments` and `list-trials`).

Reply:
Updated!